src/api/python/_api.py

```
@@ -249,6 +253,10 @@ def __init__(
                         init["thread_count"] = str(nixl_conf.num_threads)
                     elif bknd == "UCCL":
                         init["num_cpus"] = str(nixl_conf.num_threads)
+                if (bknd == "UCX" or bknd == "OBJ") and (
```

Ruff (0.15.15)

[warning] Consider merging multiple comparisons: `bknd in {"UCX", "OBJ"}`.

Merge multiple comparisons

(PLR1714)